agent/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,9 +1,9 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
 org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=11
```

Comment:
Note that this will be changed back by @Josh-Matsuoka once the JDK 8 fixes are in. :)